chapter-05/layered/repository/envelope.go

```diff
@@ -0,0 +1,94 @@
+package repository
+
+import (
+	"fmt"
+
+	"github.com/ablqk/tiny-go-projects/chapter-05/layered/money"
+)
+
+type Envelope struct {
```

Comment:
I don't think we need to expose all these types (but we need to expose the internal fields)

Reply:
Fixed.